BR: refine BR docs (#8628)
pingcap/docs: br/backup-storage-azblob.md

#### Restore

> Based on the way `account-name` is specified, you can restore data either using URLs or command-line parameters:

Reviewer: ```suggestion
Similar to how `account-name` is specified in [Back up](#back-up), you can restore data either using URLs or command-line parameters:
```

> - `account-name` is specified in URL parameters:

Reviewer: ```suggestion
- Specify `account-name` as a parameter in URL:
```

    ```
    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1'
    ```

> - `account-name` is specified in command-line parameters:

Reviewer: ```suggestion
- Specify `account-name` as a command-line parameter:
```

    ```
    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1
    ```

### Method 2: Back up and restore using an access key (easy)

> Compared with backing up and restoring data using Azure AD, backup and restoration using an access key is easier because you do not need to configure environment variables. Other steps are similar to those using Azure AD.

Reviewer: ```suggestion
Compared with backing up and restoring data using Azure AD, backup and restoration using an access key is easier because you do not need to configure environment variables. Other steps are similar to those of using Azure AD.
```

#### Back up

> - `account-name`, `account-key`, and `access-tier` are specified in URL parameters:

Reviewer: ```suggestion
- Specify `account-name`, `account-key`, and `access-tier` as parameters in URL:
```

    ```
    tiup br backup db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1&account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==&access-tier=Cool'
    ```

> - `account-name`, `account-key`, and `access-tier` are specified in command-line parameters:

Reviewer: ```suggestion
- Specify `account-name`, `account-key`, and `access-tier` as command-line parameters:
```

    ```
    tiup br backup db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1 --azblob.account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw== --azblob.access-tier=Cool
    ```

#### Restore

> - `account-name` and `account-key` are specified in URL parameters:

Reviewer: ditto. Start with a verb.

    ```
    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1&account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
    ```

- `account-name` and `account-key` are specified in in command-line parameters:

    ```
    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1 --azblob.account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
    ```

## Compatibility

This feature is **only compatible** with v5.4.0 and later versions.

## See also

> - To know other external storages supported by BR, see [External storages](/br/backup-and-restore-storages.md).

Reviewer: ```suggestion
- To learn other external storages supported by BR, see [External storages](/br/backup-and-restore-storages.md).
```

> - To know more about the parameters, see the following documents:

Reviewer: ```suggestion
- To learn more about the parameters, see the following documents:
```
